test/encryptor_test.rb

```diff
@@ -10,4 +10,14 @@ def test_encrypt_decrypt
 
     assert_equal plaintext, encryptor.decrypt(encrypted)
   end
+
+  def test_decrypt_encoding
+    encryptor = Culqi::Encryptor.new
+    plaintext = 'test string'
+    encrypted = encryptor.encrypt(plaintext)
+
+    decrypt_text = encryptor.decrypt(encrypted)
+
+    assert_equal decrypt_text.encoding.name, 'UTF-8'
```

Comment:
hmm, just to make sure the re-encoding process has not modified the content I will add another assertion:

```
assert_equal decrypt_text, 'test string'
```

You may want to use an string that have some special characters in order to make sure the text survives the re-encoding process.